gateway-developer-plugin/src/test/resources/example-project-generating-environment/src/main/gateway/config/encass.yml

```diff
@@ -1,6 +1,7 @@
 encass_env:
   annotations:
     - type: "@bundle"
```

Comment:
make this annotation simpler.
type: "@bundle" --> - "@bundle"

Reply:
@graju256 done!